src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -84,6 +85,10 @@ public static Task<RazorLanguageServer> CreateAsync(Stream input, Stream output,
                     // I think that would mean either having 0 Serial Handlers in the whole LS, or making VSLanguageServerClient handle this more gracefully.
                     .WithContentModifiedSupport(false)
                     .WithSerializer(Serializer.Instance)
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+                    .OnStarted(async (server, cancellationToken) =>
```

Comment:
You can actually implement the `IOnLanguageServerStarted` interface on any hander, and that will let that handler hook into the startup event.

Reply:
For reference: https://github.com/OmniSharp/csharp-language-server-protocol/blob/69ff46bcc8424ef9eb4d8bb566bdd3334d79e08b/src/Server/Configuration/DidChangeConfigurationProvider.cs#L83

Reply:
Oooo that's super fancy

Reply:
Neat!